example/cumulus-tf/variables.tf

```diff
@@ -350,3 +350,9 @@ variable "ems_deploy" {
   type        = bool
   default     = true
 }
+
+variable "ecs_custom_sg_ids" {
+  description = "User defined security groups to add to the Core ECS cluster"
+  type = list(string)
+  default = [""]
```

Comment:
Nitpick: do we need to initialize with a list of empty strings? I think in most cases of a variable list we just use `[]` as the default

Reply:
(not a nit if holds up approval, but good feedback)

Good catch: be64f3250